app/lib/dttp/params/placement_assignment.rb

```diff
@@ -127,6 +127,19 @@ def academic_year
         return ACADEMIC_YEAR_2021_2022 if trainee.course_start_date.between?(Date.parse("1/8/2021"), Date.parse("31/7/2022"))
         return ACADEMIC_YEAR_2022_2023 if trainee.course_start_date.between?(Date.parse("1/8/2022"), Date.parse("31/7/2023"))
       end
+
+      def training_route
+        return trainee.training_initiative if trainee.future_teaching_scholars?
+
+        trainee.training_route
+      end
+
+      def dttp_recognised_initiative?
+        [
+          ROUTE_INITIATIVES_ENUMS[:no_initiative],
+          ROUTE_INITIATIVES_ENUMS[:future_teaching_scholars],
+        ].exclude?(trainee.training_initiative)
```

Comment:
Wondering if we would benefit from switching to an `include?` instead, since hyphothetically, if that newer initiatives are added, they could get automatically allowed through in the current version.

Reply:
yeah ok! will change 👍